install/kots/manifests/gitpod-kots-config.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+# Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gitpod-kots-config
+  labels:
+    app: gitpod
+    component: gitpod-installer
+data:
+  # General settings
```

Comment:
This looks good overall on a macro level, but my main concern is that these options are now publicly visible right? esps the registry, database values being here?

Can we use a secret instead, and then load the env in the `gitpod-installer-job` using a `secretReg`? :thinking: WDYT?

Reply:
AHHHH, Right! They aren't changed. They follow the existing method of loading through secrets! 👍🏼

Reply:
Yeah, I've tried to avoid changing anything significantly in this PR - that comes later :)